File: trunk/NAI/Surface/NAI/Communication/MessageLayer/Messages/Outgoing/PairingCodeAcceptedMessage.cs

namespace NAI.Communication.MessageLayer.Messages.Outgoing
{
    internal sealed class PairingCodeAcceptedMessage : OutgoingBodyLessMessage
    {
        public PairingCodeAcceptedMessage()
            : base(MessageProtocol.PAIRING_CODE_ACCEPTED)
        { }
    }
}

File: trunk/NAI/Android/src/dk/itu/nai/comm/messages/incomming/PairingCodeAcceptedMessage.java
package dk.itu.nai.comm.messages.incomming;

import dk.itu.nai.comm.Protocol;

public class PairingCodeAcceptedMessage extends IncomingBodyLessMessage {

	public PairingCodeAcceptedMessage() {
		super(Protocol.PAIRING_CODE_ACCEPTED);
	}
}
